baiocas/client.py

```diff
@@ -36,16 +36,16 @@ class Client(object):
 
     MINIMUM_BAYEUX_VERSION = '0.9'
 
-    def __init__(self, url, io_loop=None, **options):
+    def __init__(self, url, **options):
 
         # Set up a logger for the client
         self.log = logging.getLogger('%s.%s' % (self.__module__, self.__class__.__name__))
 
         # Save the URL
         self._url = url
 
-        # Use the default IO loop if not provided
-        self.io_loop = io_loop or IOLoop.instance()
+        # Use the default IO loop
+        self.io_loop = IOLoop.current()
```

Comment:
Is this used anywhere since it's not going into `LongPollingTransport`?

Reply:
`self.io_loop` is used later in the Client class on lines 161 and 210.

Reply:
I wonder if that can be refactored too, but no need to figure that out now.